### ISSUE TITLE
Workflow

### ISSUE THREAD

**Issue by AdriVanHoudt** [iojs/iojs-nl#6]
Hi, 

I just wanted to start a discussion on how we will do things. This includes where to write the translations, what to translate, ...
Just so we can have a standard unified way of working.

Personal thoughts:
- I think posting to medium is good, other io.js members also use it and it just a good blogging tool in general
- I think every post that written by io.js or it's core members and shared via their twitter should be considered to be translated
- For collaboration we could use gists or maybe a markdown file per post in this repo for easy collab

Let me know what you think @timdp @yoshuawuyts 

**EDIT**
## Decided
- Every blog post has it's own translation file
- Every blog post is translated in Markdown style
- Every translated blog post is posted to Medium
- Prefix all blog posts files with a date
- Tag each issue/PR with either `post i18n` or `general i18n` if it is about translating a post/changelog/...
## To be decided
- What date to prefix with (posted/translated)
- 1 branch per post
- Add reference to original post/article/changelog to each translated document
- Who posts to medium and at what time

**Comment by timdp** [comment]
Yeah, I quite like Medium. It doesn't support Markdown though, so it'll be somewhat difficult to have version control with drafts as Gists.

I wonder if there's a general course of action that applies to all localized repos yet? It wouldn't make sense to come up with an entirely new one for every locale.

**Comment by AdriVanHoudt** [comment]
@timdp true medium doesn't support markdown :( you can however share drafts in medium so we can do that although it would not be version controlled :/

@mikeal do you have an idea of a general workflow that we can use or use as a base?

**Comment by AdriVanHoudt** [comment]
@timdp apparently other repos also have a gitter. I could add one to ours. I have it open all day anyway :D

**Comment by yoshuawuyts** [comment]
**RE medium:** there's an import function on medium that does accept markdown, writing the posts in gists / git is very much possible.

I'm not a fan of gitter, but if you guys want to set one up then feel free to do so. Don't think I'll be spending my time there, since the IRC channel is very much active already.

**Comment by AdriVanHoudt** [comment]
**medium:** does medium support all markdown features? And if it is possible a file per post makes sense to me. 

**gitter:** It is just something extra for instant response, if it is a big issue it will be posted here. And what IRC channel?

**Comment by yoshuawuyts** [comment]
come on over to `#io.js` on freenode. It's quite lively there (though not localized, but hey we're talking English up in this). And medium does some weird magic where they recognize bold text as being bold, links as being links, etc. etc. Just need to make sure you're reading out the html page and not the `RAW` content.

**Comment by AdriVanHoudt** [comment]
Not an IRC fan :P and i've never posted to medium so i'll let you do that :D

**Comment by bnoordhuis** [comment]
FWIW, both me and Bert hang out in the IRC channel.

**Comment by AdriVanHoudt** [comment]
@bnoordhuis cool, we'll know where to find you. Also there is [this](https://irc.gitter.im/)

**Comment by AdriVanHoudt** [comment]
[io.js-cn](https://github.com/iojs/iojs-cn) did what i proposed, adding a .md for each article. Even though not translated yet they copy(at minimum) a reference to the articles which seems like a good idea

**Comment by timdp** [comment]
I didn't know Medium had a Markdown import option. Cool!

Yeah, let's create a .md file per post then. Let's also prefix them with YYYY-MM-DD or something, so we can easily order them.

**Comment by AdriVanHoudt** [comment]
I like the prefix date.
Should we also keep the original or is that too much? If not a folder per article can be made.

Medium about markdown medium import https://medium.com/@andymcfee/how-to-import-markdown-into-medium-c06dc981bd96

**Comment by timdp** [comment]
I wouldn't keep the originals, but if you know a reason to do so, I'm certainly not opposed to it.

**Comment by AdriVanHoudt** [comment]
Not directly, was just an idea

**Comment by AdriVanHoudt** [comment]
I updated the main post for reference

**Comment by AdriVanHoudt** [comment]
Maybe add labels for blog post/article/changelog(general stuff) and non-i18n issues to keep the issues manageable?

**Comment by timdp** [comment]
Feel free to add the ones you proposed. They sound good to me, although I'm not sure if there's currently a difference between blog post and article.

**Comment by AdriVanHoudt** [comment]
I'll make a general and and a post one for now

**Comment by AdriVanHoudt** [comment]
Added some more suggestions in the original post
